.github/workflows/ci.yml

```
@@ -24,9 +55,11 @@ jobs:
         run: mix deps.get
       - name: Run Tests
         run: mix test
-  mix_format:
-    name: mix format (Elixir 1.10.2 OTP 22.3)
+  mix_check_version:
+    name: Check version (Elixir 1.10.2 OTP 22.3)
     runs-on: ubuntu-16.04
+    needs: [mix_format, mix_credo, mix_test]
+    if: github.ref != 'refs/heads/master'
```

WDT about:

```suggestion
    if: github.ref == 'refs/heads/master' && github.event == 'pull_request'
```
Run just for PRs to master branch

---

I hope that the changes will work @Danielwsx64, now the check version job will only run for branches different than master, when the event is pull request and the branch to merge is master. So only PRs to master. `github.ref != 'refs/heads/master' && github.event_name == 'pull_request' && github.base_ref == 'master'`

WDYT?

---

perfect :+1: